### DISCUSSION TITLE
pb led and ssh

### DISCUSSION THREAD

**Discussion by cirdec321** [onlaj/Piano-LED-Visualizer#491] [General]
hello

i ve begin learn piano (sorry for my English, I m French) 

i have a PI4 + LED 144Led/m   and.i ve installer the releases 1.5 on my card sd

I go in visualizer.local --> it work.  but my LED don't work 

i ve DOREMidi device between my piano (Kong 380) and my Pi4.   (just led USB and MIDI IN is ON -  Led MIDI OUT is off.  normal ?)

I can send the music of the visualizer web --> piano   but no led who work

I cannot connect too in ssh    (ssh pi@IP.  password --> raspberry) --> permission denied

**Comment by onlaj** [comment]
Hello.
Check your wiring and run https://github.com/onlaj/Piano-LED-Visualizer/blob/master/tests/strandtest.py test
On the preconfigured image the credentials are different:
Login: plv
Password: visualizer

**Comment by cirdec321** [comment]
Yeah. ssh work this that

1/I ve change the files for the GPIO PIN LED --> I put 12 (not 18). --> My led ms is on now but just mode learning. I choose a music —> learning and I have blue et green light who work on my leds or red light if note is false (and all works to mode learning)

But when I push piano key —> no respond on my led (led not illuminate)  

the backlight works so my leds works


2/ how save the configuration LED because the default configuration loaded each times
